tests/test_registration.py

```diff
@@ -9,8 +11,55 @@ def test_otp_submission():
     otp.set("1234").submit()
     assert get_url() == "http://egov-micro-dev.egovernments.org/app/v3/citizen"
 
+
 def test_create_complaint():
     comp = AddComplaintPage
     comp.set_complaint_type("garbage", "Overflowing Garbage Bins")
 
-    assert get_url() == "http://egov-micro-dev.egovernments.org/app/v3/citizen/add-complaint"
\ No newline at end of file
+    assert get_url() == "http://egov-micro-dev.egovernments.org/app/stv3/citizen/add-complaint"
+
+
+def test_login():
+    LoginPage.navigate().set("1234567890").submit()
+    assert get_url() == "http://egov-micro-dev.egovernments.org/app/v3/citizen"
+    quit_driver()
```

Comment:
`quit_driver` will not be part of the tests. I have added a new hook which will automatically quit the driver